examples/packet_mirroring/main.tf

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_providers {
+    google = {
+      version = ">= 3.45.0"
+    }
+    null = {
+      version = ">= 2.1.0"
+    }
+  }
+}
+
+resource "google_compute_network" "default" {
+  project                 = var.project_id # Replace this with your project ID in quotes
+  name                    = "new-network"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "default" {
+  project       = google_compute_network.default.project
+  name          = "my-subnet"
+  ip_cidr_range = "10.2.0.0/16"
+  region        = "us-central1"
+  network       = "new-network"
+}
+
+resource "google_compute_instance" "mirror" {
+  project      = google_compute_network.default.project
+  zone         = "us-central1-a"
+  name         = "my-instance"
+  machine_type = "e2-medium"
+
+  boot_disk {
+    initialize_params {
+      image = "debian-cloud/debian-9"
+    }
+  }
+
+  network_interface {
+    subnetwork = google_compute_subnetwork.default.id
+    access_config {
+    }
+  }
+}
+
+resource "google_compute_health_check" "default" {
+  project            = google_compute_network.default.project
+  name               = "my-healthcheck"
+  check_interval_sec = 1
+  timeout_sec        = 1
+  tcp_health_check {
+    port = "80"
+  }
+}
+
+resource "google_compute_region_backend_service" "default" {
+  project       = google_compute_network.default.project
+  region        = "us-central1"
+  name          = "my-service"
+  health_checks = [google_compute_health_check.default.id]
+}
+
+resource "google_compute_forwarding_rule" "default" {
```

Comment:
is this part necessary?

Reply:
Yes, because:

```
collector_ilb {
    url = google_compute_forwarding_rule.default.id # Replace this with a reference to your forwarding rule
  }
```

Reply:
Ack, thanks.